### PR TITLE
Enable localConnector-1.0 for InstantOn

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

File: dev/io.openliberty.checkpoint_fat/com.ibm.ws.jmx.fat.jmxtest.bnd [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#*******************************************************************************
```
Comment (dazavala) [inline dev/io.openliberty.checkpoint_fat/com.ibm.ws.jmx.fat.jmxtest.bnd:1]: Update copyright year

**Review by dazavala — CHANGES_REQUESTED** [review]
Looking GTM except for that one copyright date.

**Comment by tjwatson** [comment]
#build

**Comment by dazavala** [comment]
#libby

**Review by dazavala — APPROVED** [review]
LGTM. Hailed libby for you.

**Pull request merged**